src/pages/Home.jsx

```diff
@@ -168,26 +171,18 @@ const Home = () => {
     }})
     remove()
   }
-
+  
   useEffect(() => {
-    if (contracts.MarketContract) {
-      // loadListings();
-    }
-  }, [contracts.MarketContract, filters, sortBy]);
+    const token = PubSub.subscribe("LISTING_EVENT", (msg, data) => {
+      queryClient.invalidateQueries('listings')
```

Comment:
will this refetch previously fetched listings? Because a user might've already fetched 3 pages of listings before getting a new listing event. In that case will the home page be in loading state again?

Reply:
I have the same question as Michael.